config.yaml

```diff
@@ -5,9 +5,9 @@ receivers:
   otlp:
     protocols:
       grpc:
-        endpoint: 0.0.0.0:55680
+        endpoint: 0.0.0.0:4317
       http:
-        endpoint: 0.0.0.0:55681
+        endpoint: 0.0.0.0:4318
```

Comment:
It's weird the upstream haven't changed the default port for http protocol yet - https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/otlpreceiver/factory.go#L43

Reply:
@mxiamxia It is a good catch. There is an open issue for splitting ports of OTLP receiver. I updated this PR to only change 55680 to 4317 for OTLP/gRPC before the issue is resolved in upstream. 
https://github.com/open-telemetry/opentelemetry-collector/issues/1256